using_images/s2i_images/customization.adoc

```diff
@@ -0,0 +1,84 @@
+[[using-s2i-images]]
+= Using S2I images
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+
+S2I builder images normally include
+xref:../creating_images/s2i.html#s2i-scripts[*_assemble_* and *_run_* scripts],
+but the default behavior of those scripts may not be suitable for all users.
+This document will cover a few approaches for customizing the behavior of an
+S2I builder that includes default scripts.
+
+== Invoking scripts embedded in an image
```

Comment:
Headings need to be in title case and an anchor needs to go immediately before, so:

[[using-S2I-images-invoking-scripts-embedded-in-an-image]]
== Invoking Scripts Embedded in an Image